domains/jannik.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "janniksohn",
+        "email": "qeufdsvkngstspdlihbq6k89pcwuhc@janniksohn.com"
+    },
+    "records": {
+        "A": ["216.198.79.1"]
```

```suggestion
        "CNAME": "cname.vercel-dns.com"
```

not a requirement but i recommend you use cname